src/main/java/uk/gov/ons/ctp/response/notify/service/impl/ConfigurationAwareNotificationClient.java

```diff
@@ -0,0 +1,97 @@
+package uk.gov.ons.ctp.response.notify.service.impl;
+
+import uk.gov.ons.ctp.response.notify.config.NotifyConfiguration;
+import uk.gov.service.notify.*;
+
+import java.util.Map;
+
```

Comment:
Should add a logger to this class. It the least it should log at debug level if it's not passing method calls onto the composite class

Reply:
Will do

Reply:
Done